src/app/pages/swap/hooks/use-bitflow-swappable-assets.tsx

```diff
@@ -70,11 +70,11 @@ function useCreateSwapAsset(address: string) {
         ...swapAsset,
         balance: availableBalance ?? createMoney(0, token.symbol, token.tokenDecimals),
         marketData: availableBalance
-          ? priceAsMarketData(swapAsset.principal, availableBalance.symbol)
-          : priceAsMarketData(swapAsset.principal, token.symbol),
+          ? getTokenMarketData(swapAsset.principal, availableBalance.symbol)
```

Comment:
Aren't `token.symbol` and `availableBlance.symbol` the same? Thinking that because of this logic:

``` 
const availableBalance = sip10Tokens.find(
        sip10Token => getPrincipalFromContractId(sip10Token.info.contractId) === token.tokenContract
      )?.balance.availableBalance;
```

Where the contract info is matched on the `token.tokenContract`